tools/mtouch/Assembly.cs

```diff
@@ -94,6 +94,26 @@ public void CopyMdbToDirectory (string directory)
 				Application.UpdateFile (mdb_src, mdb_target);
 			}
 		}
+		
+		public void CopyMSymToDirectory (string directory)
+		{
+			string msym_src = FullPath + ".aotid.msym";
+			var dirInfo = new DirectoryInfo (msym_src);
+			if (!dirInfo.Exists) // got no aot data
+				return;
+			var subdirs = dirInfo.GetDirectories();
+			foreach (var subdir in subdirs) {
+				var destPath = Path.Combine (directory, subdir.Name.ToUpper ());
```

Comment:
Not required per se, the issue is that the AOTID is lower case and the MVID is upper case. I would prefer to have both on the same style. If you see that as a must, I'll use Application.UpateDirectory.